menu.js

```diff
@@ -20,3 +21,211 @@ navMenu.addEventListener('click', () => {
+    workSectionContainer.appendChild(popupWindowMainCont);
+    popupWindowMainCont.className = 'popupWindowCont';
+    const popupwindow = document.createElement('section');
+    popupWindowMainCont.appendChild(popupwindow);
+    popupwindow.className = 'popupWindow';
+    const closeIcon = document.createElement('img');
+    closeIcon.src = 'images/xIcon.svg';
+    closeIcon.style.float = 'right';
+    popupwindow.appendChild(closeIcon);
+    closeIcon.addEventListener('click', () => {
+      languagesUlLi4.style.display = 'none';
+      languagesUlLi5.style.display = 'none';
+      languagesUlLi6.style.display = 'none';
+      popupWindowMainCont.style.display = 'none';
+      bgdiv.appendChild(h2);
+      bgdiv.appendChild(bgdivUl);
+      cardSection.appendChild(featuredImage);
+      bgdiv.appendChild(bgdivP);
+      bgdivP.textContent = popupObject[i].description;
+      bgdiv.appendChild(languagesUl);
+      bgdiv.appendChild(btn);
+      btn.style.width = '130px';
+      languagesUl.style.marginBottom = '0';
+    });
+    popupwindow.appendChild(h2);
+    popupwindow.appendChild(bgdivUl);
+    popupwindow.appendChild(featuredImage);
+    const pDivider = document.createElement('div');
+    pDivider.className = 'pDivider';
+    popupwindow.appendChild(pDivider);
+    pDivider.appendChild(bgdivP);
+    bgdivP.textContent = popupObject[i].popupDescription;
+    const techDiv = document.createElement('div');
+    techDiv.className = 'techDiv';
+    pDivider.appendChild(techDiv);
+    techDiv.appendChild(languagesUl);
+    const popupbtn1 = document.createElement('a');
+    const popupbtn2 = document.createElement('a');
+    const popupbtnDiv = document.createElement('div');
+    techDiv.appendChild(popupbtnDiv);
+    popupbtn1.className = 'popupClickme1';
+    popupbtn2.className = 'popupClickme2';
+    popupbtn1.textContent = 'See live';
+    popupbtn2.textContent = 'See source';
+    const icon1 = document.createElement('img');
+    const icon2 = document.createElement('img');
+    icon2.src = 'images/githubinbutton.svg';
+    popupbtn2.appendChild(icon2);
+    icon1.src = 'images/roundShape.png';
+    icon1.style.marginBottom = '-3px';
+    icon2.style.margin = '0 5px -3px 6px';
+    icon1.style.margin = '0 5px -3px 6px';
+    popupbtn1.appendChild(icon1);
+    popupbtn2.href = popupObject[i].secodbutton;
+    popupbtnDiv.append(popupbtn1, popupbtn2);
+    const hr = document.createElement('hr');
+    hr.className = 'hr';
+    techDiv.insertBefore(hr, popupbtnDiv);
+    hr.style.color = '#ebecf0';
+    popupbtnDiv.className = 'popupBtnDiv';
+    languagesUl.style.marginBottom = '20px';
+  });
+}
+
+// form validation//
+const form = document.querySelector('#contactform');
+const email = document.querySelector('#mail');
+const invalidError = document.querySelector('#invalidError');
+
+form.addEventListener('submit', (e) => {
+  const emailText = email.value;
+  if (emailText !== emailText.toLowerCase()) {
+    e.preventDefault();
```

Comment:
- [ Optional] Great job on implementing the validation form, but I think you can do better if you add the email pattern, as you can see in the image below, the user can submit any kind of content in the email field, please consider that.
- 
<img width="350" alt="emailpat" src="https://user-images.githubusercontent.com/79058364/157979813-25886baf-a9b2-4ca9-9e0e-e166b1b10b84.png">

- [OPTIONAL ] Please Adjust the color of your validation error, I think this red does not really match your design
<img width="349" alt="color" src="https://user-images.githubusercontent.com/79058364/157979973-9838ce9b-6a18-4785-ab9b-17946890bef7.png">